candi/bashible/common-steps/all/004_check_hostname.sh.tpl

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Flant JSC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function check_hostname(){
+    local a=`hostname`
```

Comment:
Could you use parenthesis `$()` please. Backticks are more inconspicuous and couldn't be nested. So, it's not an error but it's harder to read

Reply:
also we use construction $(hostname -s) everywhere in code.